tests/lib/rook/base.py

```diff
@@ -92,8 +97,16 @@ def install(self):
             matcher=common.regex_count_matcher(pattern, 1),
             attempts=30, interval=10)
 
-        self.kubernetes.kubectl_apply(
-            os.path.join(self.ceph_dir, 'cluster.yaml'))
+        logger.info("Rook successfully installed and ready!")
+
+    def install_ceph(self, cluster_yaml=None, health_test=True):
+        if cluster_yaml is None:
+            self.kubernetes.kubectl_apply(
+                os.path.join(self.ceph_dir, 'cluster.yaml'))
+        else:
+            self.kubernetes.kubectl_apply(
+                os.path.join(self.ceph_dir, cluster_yaml))
```

Comment:
(nit) Why not just give a default parameter?
`
def install_ceph(self, cluster_yaml='cluster.yaml', health_test=True):
`

Reply:
Good point